app/views/accounts/show.html.erb

```diff
@@ -35,6 +36,7 @@
               </div>
               <div class="transactions-attr"><%= temp %><%= transaction.recipient %></div>
               <div class="transactions-attr">Time: <%= transaction.time.strftime("%I:%M%p on %d/%m/%Y") %></div>
+            <% end %>
```

Comment:
if the transaction amount is nil then we dont show it, £0 will still show curently